frontend/src/lib/taxonomy.tsx

```diff
@@ -1324,6 +1406,70 @@ CORE_FILTER_DEFINITIONS_BY_GROUP.event_properties.$session_duration =
     CORE_FILTER_DEFINITIONS_BY_GROUP.session_properties.$session_duration
 
 export const PROPERTY_KEYS = Object.keys(CORE_FILTER_DEFINITIONS_BY_GROUP.event_properties)
+export const NON_DOLLAR_POSTHOG_PROPERTY_KEYS = [
```

Comment:
Should these be dollared? I don't think they're _strictly_ "PostHog" properties in the same way `$` properties are. They look like properties we track within our own implementations of PostHog e.g. a customer would never have a `projected_usage.data_warehouse` property on their events

Reply:
Good point

Reply:
So, I didn't add them to the taxonomy since they're not posthog posthog properties

However, I never want to see the billing properties. They make it basically impossible to see what properties are actively added to an event

So, if on posthog cloud or on dev we hide these properties as if they're posthog properties

Reply:
```suggestion
// these are properties that PostHog add to events they track for their own purposes
// not part of the general taxonomy
// but often more numerous than actual properties set on events and useful to hide 
// to make those properties discoverable
export const NON_DOLLAR_POSTHOG_PROPERTY_KEYS = [
```

Reply:
does the comment help?